index.html

```diff
@@ -0,0 +1,132 @@
+            <ul class="nav-content">
+                <li><a href="about.html">About</a></li>
+                <li><a href="#program" id="nav-grey">Program</a></li>
+                <li><a href="#join" id="nav-grey">Join</a></li>
+                <li><a href="#sponsor" id="nav-grey">Sponsor</a></li>
+                <li><a href="#news" id="nav-grey">News</a></li>
+            </ul>
+        </nav>
+
+    </header>
+
+    <section class="intro">
+        <p id="hello">"Hello!,Sharing world"</p>
+        <div class="title-intro">
+            <h1>FORMULA 1 FANS GLOBAL SUMMIT 2023</h1>
+        </div>
+        <div class="p-intro">
+            <p class="summit-intro">Welcome to the 7th global summit of Formula 1 fans.
+                Celebrate,have fun and bless someone.Venue:Vienna,Austria. </p>
+        </div>
+        <div class="date">
+            <p class="show-date">2023.9.8(FRI) ~ 9(SAT)</p>
+            <p>@Grand Prix of Vienna and more</p>
+        </div>
+    </section>
+
+    <section class="program">
+        <h2>Main program</h2>
+        <hr class="sm-line">
+
+        <div id="gr-program">
+            <div class="grid-program">
+                <img src="images/program_icon_01 1.svg" alt="lecture-icon" class="program-icon">
+                <p class="program-points">Lecture</p>
+                <p class="program-ex">Listen to the speakers from various countries 
+                    about the messages of sharing and opening.</p>
+            </div>
+
+            <div class="grid-program">
+                <img src="images/program_icon_02 1.svg" alt="exhibition-icon" class="program-icon">
+                <p class="program-points">CC Exhibition</p>
+                <p class="program-ex">Appreciate various creations applying 
+                    CC license of artists, organized from Art Center Nabi</p>
+            </div>
+
+            <div class="grid-program">
+                <img src="images/program_icon_03 1.svg" alt="forum-icon" class="program-icon">
+                <p class="program-points">Forum</p>
+                <p class="program-ex">Have the time to share your thoughts and 
+                    opinions with experts for each topic.</p>
+            </div>
+
+            <div class="grid-program">
+                <img src="images/program_icon_04 1.svg" alt="workshop-icon" class="program-icon">
+                <p class="program-points">Workshop</p>
+                <p class="program-ex">Try creating your own work using open source license 
+                    rather than just watching at it.</p>
+            </div>
+
+            <div class="grid-program">
+                <img src="images/program_icon_05 1.svg" alt="ignite-icon" class="program-icon">
+                <p class="program-points">CC ignite</p>
+                <p class="program-ex">get opportunities to network with CC affiliates around the world, 
+                    also after the summit.</p>
+            </div>
+        </div>
```

Comment:
- [ ] Good job on the main section 😄 , but it could be better if all the content sticks to the Cindy design 😄 . The spacing should stick too 

- [ ] Make sure you implement the `hover` selector as it is visible 👍 in the Cindy

### Your design

![Capture4](https://github.com/MarkYegon7/Capstone-Project/assets/80458199/c3ec80ae-00c8-4d3c-b47c-1a400fc3e956)

### Cindy Design

![Capture33](https://github.com/MarkYegon7/Capstone-Project/assets/80458199/204e0eac-16cf-4cab-8a11-cf84ef78aae1)

Reply:
Fixed spacing and added hover effect